src/components/FindPark.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <section>
+    <h2>Let's Go Play!</h2>
+    <button @click="searchByLocation">Find a dog park near me!</button>
+    <h3>--Or--</h3>
+    <input
+      class="search-input"
+      type="text"
+      placeholder="Search by name, city or zip code"
+      ref="enteredValue"
+    />
+    <button class="search-button" @click="setSearchTerm">
+      Get Started - woof!
+    </button>
+    <p v-if="inputIsInvalid">Please enter a park name, city, or zip code</p>
+  </section>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      searchTerm: '',
+      inputIsInvalid: false,
+    };
+  },
+  methods: {
+    setSearchTerm() {
```

Comment:
I love what you did here with `setSearchTerm( )`.  Until now, I hadn't been aware of `$refs`, so I did some research [here](https://v3.vuejs.org/guide/component-template-refs.html).  This seems pretty similar to `event` where you could use `event.target.value` as an assigned value for `enteredValue`.  

I'm curious, did you find a benefit to using `$refs` rather than `event` ?  I'd love to know more!